examples/vic-gov-au/test/e2e/fixtures/snippets/blockquote.html

```diff
@@ -0,0 +1 @@
+<blockquote class="quotation"><p>Mauris tincidunt tincidunt felis vel tempus. Vestibulum rhoncus blandit justo quis finibus. Phasellus lacus lectus, sollicitudin sed posuere non, ultricies ut quam.</p> <footer><cite><span class="quotation__author">Duis ligula lacus</span><br> <span class="quotation__author-title">Phasellus est turpis, efficitur nec odio imperdiet</span><br></cite></footer></blockquote>
```

Comment:
Add seperate fixture for each wysiwyg feature, try to work around issue with month in asset URLs